app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -111,6 +111,7 @@ def parse_vm(vm)
     status        = vm.status
     uid           = vm.id
     name          = vm.name
+    hostname      = vm.customization.computer_name
```

Comment:
Can you handle if `vm.customization` is nil?  Either `vm.customization.try(:computer_name)` or `vm.customization.computer_name if vm.customization`